src/main/resources/swagger.api/schemas/marcFormat.json

```diff
@@ -6,6 +6,7 @@
   "additionalProperties": false,
   "enum": [
     "BIBLIOGRAPHIC",
-    "HOLDINGS"
+    "HOLDINGS",
+    "AUTHORITY"
```

Comment:
You should bump minor interface version in case of changing schemas

Reply:
Upgrade `records-editor.yaml` and `records-editor-async.yaml` versions to 1.1.1 or 1.2.0?